src/main/java/redis/clients/jedis/Connection.java

```diff
@@ -122,8 +127,10 @@ public void sendCommand(final ProtocolCommand cmd) {
 
   public void sendCommand(final ProtocolCommand cmd, final byte[]... args) {
     try {
+      listener.commandStarted(this, cmd, args);
       connect();
```

Comment:
`listener.commandConnected(this, cmd)` would be helpful here to have the option of tracking connection time separately/in addition to command execution time.